exercises/pangram/examples/success-text/src/Pangram.hs

```diff
@@ -0,0 +1,11 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Pangram (isPangram) where
+
+import qualified Data.Text as T
+import           Data.Text (Text)
+
+isPangram :: Text -> Bool
+isPangram s = T.all (`member` T.toLower s) alphabet
+  where
+    alphabet = "abcdefghijklmnopqrstuvwxyz"
+    member = T.any . (==)
```

Comment:
I don't like this definition particularly much, but it seems that `Data.Text` doesn't have a less general membership function, and since `Text` isn't `Foldable`, I can't use `elem`.